articles/connections/database/password-change.md

```diff
@@ -65,18 +68,25 @@ If you're calling this from the browser, don't forget to add your URL to the `Al
 If your Connection is a custom database and the user exists in the database, invoke the Authentication API for `changePassword`.
 :::
 
-If the `POST` call is successful, the user receives an email containing a link to reset their password.
+Once the password reset has been triggered, the user will now [receive a password reset email](#password-reset-emails).
 
-![](/media/articles/connections/database/reset-password-email.png)
+### Use Lock
```

Comment:
Do we need to mention Lock here? In the context of avoiding talking about Lock / Embedded login, unless it's the specific Embedded Login section, I'd remove it.